adapters/adtrgtme/adtrgtme.go

```diff
@@ -36,54 +36,52 @@ func (v *adapter) MakeRequests(
 	var errors []error
 
 	requestCopy := *openRTBRequest
-
-	for _, imp := range openRTBRequest.Imp {
-		requestCopy.Imp = []openrtb2.Imp{imp}
-
-		requestJSON, err := json.Marshal(openRTBRequest)
+	for i := range openRTBRequest.Imp {
+		siteID, err := siteIDFromImp(&openRTBRequest.Imp[i])
 		if err != nil {
 			errors = append(errors, err)
 			continue
 		}
+		openRTBRequest.Imp[i].Ext = nil
```

Comment:
Warning that you are modifying the original openRTBRequest.Imp[i].Ext and we should not ideally be modifying the input impression. I think this for loop could be re-written as - 
```
for _, imp := range openRTBRequest.Imp {
                 siteID, err := getSiteIDFromImp(&imp)
		if err != nil {
			errors = append(errors, err)
			continue
		}
                 
                 imp.Ext = nil
                 requestCopy.Imp = []openrtb2.Imp{imp}

                 requestBody, err := json.Marshal(requestCopy)
		if err != nil {
			errors = append(errors, err)
			continue
		}
		
		requestData := &adapters.RequestData{
			Method:  http.MethodPost,
			Uri:     v.buildRequestURI(siteID),
			Body:    requestBody,
			Headers: makeRequestHeaders(openRTBRequest),
		}
		
		requests = append(requests, requestData)
}
```